tests/identities/test_trait_schema.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from flag_engine.identities.schemas import TraitSchema
+
+
+@pytest.mark.parametrize(
+    "trait_data",
+    (
+        {"trait_key": "key", "trait_value": "value"},
+        {"trait_key": "key1", "trait_value": 21},
+        {"trait_key": "key1", "trait_value": None},
+        {"trait_key": "key1", "trait_value": 11.2},
+        {"trait_key": "key1", "trait_value": True},
+    ),
+)
+def test_trait_schema_load_and_dump(trait_data):
```

Comment:
We should split this into 2 separate tests and split up the parametrized inputs to make the tests more readable: 

```python
@pytest.mark.parametrize(
    "trait_key, trait_value",
    (
         ("key", "value"),
         ("key", 21), 
         ...
    )
)
def test_trait_schema_load(trait_key, trait_value):
    # Given
    trait_schema = TraitSchema()

    # When
    trait_model = trait_schema.load({"trait_key": trait_key, "trait_value": trait_value})

    # Then
    assert trait_model.trait_key == trait_key
    assert trait_model.trait_value == trait_value


@pytest.mark.parametrize(
    "trait_model",
    (
         TraitModel("key", "value"),
         TraitModel("key", 21), 
         ...
    )
)
def test_trait_schema_load(trait_model):
    # Given
    trait_schema = TraitSchema()

    # When
    trait_data = trait_schema.dump(trait_model)

    # Then
    assert trait_data["trait_key"] == trait_model.trait_key
    assert trait_data["trait_value"] == trait_model.trait_value
```